trident/light_ray.py

```diff
@@ -451,6 +451,10 @@ def make_light_ray(self, seed=None, periodic=True,
         ...                       use_peculiar_velocity=True)
 
         """
+
+        # force unitary units to exist if they don't already
+        self.code_length.in_units("unitary")
```

Comment:
`in_units` will return a copy, but not alter the units of the existing array. For that, you'll need `convert_to_units`.

Reply:
Actually, `self` here does not have a `code_length` attribute. What's the goal of this here?

Reply:
Oh this was just me testing something. I didn't realize it had submitted this with the pull. Let me remove it.